device/device.go

```diff
@@ -200,7 +199,7 @@ func (d *Device) GetPixFormat() (v4l2.PixFormat, error) {
 		return v4l2.PixFormat{}, v4l2.ErrorUnsupportedFeature
 	}
 
-	if reflect.ValueOf(d.config.pixFormat).IsZero() {
+	if d.config.pixFormat.IsZero() {
```

Comment:
I am not convinced we need the additional method on type PixFormat, just to test for its zero-ness. Since this is only done here (in the entire code base), just do the test here:

```go
if (d.config.pixFormat == PixFormat{}) { ... } // i think this will work
```

Reply:
This works:
`if d.config.pixFormat == (PixFormat{}) { ... } `

We need to add `()` around `PixFormat{}` to satisfy the Go compiler.